### ISSUE TITLE
Bounty is closed?

### ISSUE THREAD

**Issue by Dolani25** [jaimehgb/RaiWebWallet#45]
Im guessing the bounty hunt has been closed?

**Comment by jaimehgb** [comment]
Yes. Nanowallet.io is no longer maintained (although still online for people to be able to login and export their seed).
The wallet in the readme, though, still contains 2000 Nano.

**Comment by Dolani25** [comment]
Alright🥲. Thank You for that information.

**Comment by SarixXaris** [comment]
Hi @jaimehgb. You said you will give the password format for the bounty? May I know the password format?
